packages/shared/src/buttons/LinkButton.js

```diff
@@ -11,15 +12,17 @@ import Button from './Button';
 type Props = {|
   +title: TranslationType,
   +onPress: Function,
+  +titleStyle?: StylePropType,
 |};
 
 /**
  * Button that looks like a simple link.
  */
 export default function LinkButton(props: Props) {
+  const titleStyle = props.titleStyle || null;
```

Comment:
this line is not needed
`style={[styleSheet.text, titleStyle]}` works fine even if `props.titleStyle` is undefined